.travis.yml

```diff
@@ -6,3 +6,4 @@ install:
   - pip install -r test-requirements.txt
 script:
   - flake8 .
+  - pytest ./tests/test_utils.py
```

Comment:
This is going to execute `test_utils.py` only, please make it run any test that is in the `tests` directory, so we don't have to update this line the next time we add a test.

Reply:
Don't forget about this, @Shashi456